docker/ci/Dockerfile

```
@@ -1,6 +1,12 @@
 FROM ubuntu:22.04
 
-RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get -y install ca-certificates
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
+    ca-certificates \
+    python3 \
+    python3-dev \
+    python3-pip
+
+RUN pip install pyarrow
```

```suggestion
RUN python3 -m pip install pyarrow
```

Not quite understand the best practice, but this should be more robust